### PR TITLE
*: use the `filepath` package to make unit test portable on windows

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What problem does this PR solve? 

```
fmt.Println(path.Join(os.TempDir(), "xxx.csv"))
fmt.Println(filepath.Join(os.TempDir(), "xxx.csv"))
```

```
C:\Users\genius\AppData\Local\Temp/xxx.csv
C:\Users\genius\AppData\Local\Temp\xxx.csv
```

Use `filepath` is much better, note the slash `\`, the former is not a valid path.
As I've said, I have to use windows OS for a while, this change helps me to run tests.

### Check List 

Tests 

 - No code

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @qw4990 @jackysp

File: executor/select_into_test.go [hunk 1]
```diff
@@ -60,28 +61,28 @@ func (s *testSuite1) TestSelectIntoOutfileFromTable(c *C) {
 	tk.MustExec("insert into t values (null, null, null, null, '2000-03-03', '03:03:03', '03:03:03', '[1,2,3]')")
 	tk.MustExec("insert into t values (4, 4.4, 0.4, 'd', null, null, null, null)")
 
-	tk.MustExec(fmt.Sprintf("select * from t into outfile '%v'", outfile))
+	tk.MustExec(fmt.Sprintf("select * from t into outfile %s", strconv.Quote(outfile)))
```
Comment (jackysp) [inline executor/select_into_test.go]:
```suggestion
	tk.MustExec(fmt.Sprintf("select * from t into outfile %q", outfile))
```
how about this?

File: config/config_util.go [hunk 2]
```diff
@@ -106,7 +106,7 @@ func atomicWriteConfig(c *Config, confPath string) (err error) {
 	if err != nil {
 		return err
 	}
-	tmpConfPath := path.Join(os.TempDir(), fmt.Sprintf("tmp_conf_%v.toml", time.Now()))
+	tmpConfPath := filepath.Join(os.TempDir(), fmt.Sprintf("tmp_conf_%v.toml", time.Now().Unix()))
```
Comment (jackysp) [inline config/config_util.go]:
```suggestion
	tmpConfPath := filepath.Join(os.TempDir(), fmt.Sprintf("tmp_conf_%v.toml", time.Now().Format("20060102150405")))
```
maybe more readable

**Comment by tiancaiamao** [comment]
/run-all-tests
PTAL @qw4990 @jackysp

**Pull request merged**